.ipynb_checkpoints/Untitled-checkpoint.ipynb

```diff
@@ -0,0 +1,1189 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import XYMEClient from './packages/typescript/accern_xyme'\n",
+    "import { XYMEConfig } from './packages/typescript/types'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "// const config: XYMEConfig = {\n",
+    "//     url: \"http://localhost:8080/\",\n",
+    "//     token: \"5173c3a07df24475a05d911977904b4b\",\n",
+    "//     namespace: \"default\"\n",
+    "// }\n",
+    "\n",
+    "const testConfig = {\n",
+    "    url: \"http://test-v3.accern.com/xyme\",\n",
+    "    token: \"b70ae4cc1cb94a65967e129004900a27\",\n",
```

Comment:
you know that this repo is publicly visible, right? make sure to not include those files/information!